.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
We usually use this trigger in other workflows:
https://github.com/dogtagpki/jss/blob/master/.github/workflows/required.yml#L3
Should we use the same trigger, or is there a specific reason for this?

Reply:
I cargo-culted the whole YML file, if there is a convention then I will follow it.

Reply:
Thanks for the update! It's not really a convention, but we want to run the CI on all pushes & pull requests without any branch restrictions, so that's what we've been using.